src/components/select/select.template.tsx

```diff
@@ -324,3 +324,15 @@ function getSelectedIcon(value: any) {
         />
     );
 }
+
+export function triggerIconColorWarning(values: Option | Option[]) {
+    const selectedValues = Array.isArray(values) ? values : [values];
+    for (const value of selectedValues) {
+        if (value.iconColor) {
+            /* eslint-disable-next-line no-console */
+            console.warn(
+                "The `iconColor` prop is deprecated now! Use the new `Icon` interface and instead of `iconColor: 'color-name'` write `icon {name: 'icon-name', color: 'color-name'}`."
+            );
+        }
+    }
+}
```

Comment:
This doesn't generate the console.warn for limel-select if `iconColor` is used, and I have no idea how to fix this. I tried a lot.